package.json

```diff
@@ -66,6 +66,7 @@
     "node": ">=8.9"
   },
   "dependencies": {
+    "@ledgerhq/hw-transport-node-hid": "^5.12.0",
```

Comment:
Is this going to work in the browser?

Reply:
no, Martin said to only support the desktop wallet for now. Just have to make sure it doesn't break in the browser

Reply:
In electron, you have 2 environments - the main thread is node.js, and the windows are effectively Chrome. I'm not a 100% sure, but I think the stuff running in the BrowserWindows, which this will have to be in, is effectively a browser environment. I think it makes sense to check at least

Reply:
ahh interesting ... that's very good to know. Will definitely test in the wallet environment to make sure. I know at least that some ledger transports are compatible with browser and some are for node. Just need to test it out.